demo/src/test/java/amazon/TaskFourPageObjectOneTest.java

```diff
@@ -0,0 +1,169 @@
+package amazon;
+
+import com.google.common.collect.Ordering;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class TaskFourPageObjectOneTest {
+
+    private static final String LEGO = "LEGO";
+    private static final String SPAN_CLASS_A_PRICE_WHOLE = "//span[@class='a-price-whole']";
+    private static final String URL_FOR_FILTER = "https://www.amazon.com/s?" +
+            "k=school+supplies" +
+            "&crid=3B2SMJ2SQ64N" +
+            "&pd_rd_r=002c7e59-b3e2-4870-9de7-a9c54fb9263c" +
+            "&pd_rd_w=9XU4m&pd_rd_wg=DkrTY" +
+            "&pf_rd_p=c0f33dcc-b6ee-417c-b438-e358c4a5e8dc" +
+            "&pf_rd_r=BSVWZBX1JYQGKW4WR8F4" +
+            "&sprefix=school+su%2Caps%2C175" +
+            "&ref=pd_gw_unk";
+
+    private WebDriver driver;
+
+
+    @FindBy(id = "twotabsearchtextbox")
```

Comment:
Should be moved in a separate page class

Reply:
Fixed. Added AmazonSearchFilterPriceHomePage.